index.css

```diff
@@ -0,0 +1 @@
+h1 { color: darkgreen; }
```

Comment:
- [OPTIONAL] It is a best practice to use class selector instead of element selector to style the content of your `h1` element. I marked it as optional because, it was not mentioned by the previous reviewer. Kindly attend to it.